dockerfile

```diff
@@ -8,4 +8,5 @@ COPY . .
 
 EXPOSE 8080
 
+RUN flask db upgrade
```

Comment:
I think we want to run the db upgrade in the final command, so it happens at runtime rather than build time. This is the thinking behind the formulation in https://github.com/communitiesuk/funding-service-design-assessment-store/blob/d94836488eeb9d607f5d7e0517938cd5ee01981c/Dockerfile#L14